conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - 11.0                   # [osx]
\ No newline at end of file
+c_stdlib_version:  # [osx]
+  - 11.0           # [osx]
```

Comment:
Are you sure you need this file? The package is noarch, right?

Reply:
it is noarch, the linter complained, that's why I changed it.

```
For recipes/sphinxcontrib-video:

In your conda_build_config.yaml, please change the name of MACOSX_DEPLOYMENT_TARGET, to c_stdlib_version!
```

Reply:
Do you need the deployment target? Normally noarch packages don't need any of this.

Reply:
When the solution is to remove it, I do that.

The hint could have been a bit more detailed.

Reply:
Well, the hint assumed you added that file b/c you neeed it. Our lints are not smart enough to know everything about your package. The question is, why you added that in the first place? If you don't know you need extra config, try without them first.

Reply:
This is also happening to me in #26412

Reply:
Ah. Good to know!